content/rust/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'Data Types in Rust'
+Description: 'An overview of data types in Rust, including scalar and compound types.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Rust'
+  - 'Data Types'
+CatalogContent:
+  - 'rust-for-programmers'
+  - 'paths/computer-science'
+---
+
+In Rust, data types classify the types of values that can be used and manipulated within a program. Rust's type system ensures that data types are known at compile time, which helps catch errors early.
+
+## Scalar Types
+
+Rust has four primary scalar types:
+- **Integers**: Represent whole numbers. Examples include `i32`, `u32`, `i64`, etc.
+- **Floating-Point Numbers**: Represent numbers with fractional components. Examples include `f32`, `f64`.
+- **Booleans**: Represent true or false values, using the `bool` type.
+- **Characters**: Represent single Unicode characters, using the `char` type.
+
+```rust
+fn main() {
+    let int: i32 = 42;
+    let float: f64 = 3.14;
+    let boolean: bool = true;
+    let character: char = 'R';
+
+    println!("Integer: {}", int);
+    println!("Float: {}", float);
+    println!("Boolean: {}", boolean);
+    println!("Character: {}", character);
+}
+```
+
+## Compound Types
+
+Rust also provides compound types:
+- **Tuples**: Group multiple values of different types into one compound type.
+- **Arrays**: Store multiple values of the same type.
+
+```rust
+fn main() {
+    let tuple: (i32, f64, bool) = (42, 3.14, true);
+    let array: [i32; 3] = [1, 2, 3];
+
+    println!("Tuple: ({}, {}, {})", tuple.0, tuple.1, tuple.2);
+    println!("Array: {:?}", array);
+}
+```
```

Comment:
Can you add the output for the above code?

Reply:
Sure thing. Not a "gotcha" or anything but, on the pandas tail, the output of the codebyte example was requested to be omitted. And, in this case, I assumed since the gif shows an output, it may be redundant to share an output. So I assumed to not include it. For future reference, could you clarify when to (or not to) have a the output written? Perhaps it is case-by-case. (I did not find anything discreet in the contribution guidelines)

Reply:
Hey @sethaferd! In the `pandas.tail()` PR, I asked you to add a Codebyte along with an example, but you removed one of them 🥲 So, [here](https://github.com/Codecademy/docs/pull/4855/files) is your `.tail()` PR, it consists of both an example and a codebyte block.

For term entries based on languages - javaScript, python 3, C++, C#, Go, PHP, Ruby, Scheme - we ask for a Codebyte (codebyte is not followed by an output block). However, an example is usually needed for all entries and it should be always followed by an output block. 
Hope this helps! 😃